modules/ROOT/nav.adoc

```diff
@@ -134,7 +134,7 @@ include::partial$nav-dw.adoc[]
   *** xref:cryptography-xml.adoc[XML]
   *** xref:cryptography-jce.adoc[JCE]
   *** xref:cryptography-reference.adoc[Cryptography XML Reference]
-I** xref:debugging-outside-studio.adoc[Debugging Outside Studio]
+* xref:debugging-outside-studio.adoc[Debugging Outside Studio]
```

Comment:
Leaving this with * will make it a top level, and the following articles with ** will be inside this one. 
If this article needs to be just below Mule Runtime, send this line at the bottom of the file.

Reply:
It is looking fine in the build. I will do another commit.

Reply:
Cristian, if you select the second commit it looks OK in the diff. I don't know why the View File shows something different. This is confusing.